.github/workflows/release-step-3_automatic_prepare-docker-tags.yml

```diff
@@ -5,6 +5,14 @@ on:
     tag_version:
      description: 'Semantic Version'
      required: true
+    docker_meta_tag:
+     type: choice
+     options:
+      - do_not_tag
+      - latest
+      - next
```

Comment:
what about setting a default ?

Reply:
it defaults to the first choice:
![image](https://user-images.githubusercontent.com/2575450/168043444-d5aae841-df89-4c88-b089-421e3b24821b.png)